src/views/v2/list/top-bar/datepicker.php

```diff
@@ -48,22 +51,34 @@
 $selected_start_date_label = date_i18n( $selected_start_date_label_format, $selected_start_datetime );
 $selected_end_date_label = date_i18n( $selected_end_date_label_format, $selected_end_datetime );
 
+$selected_start_date_mobile = Dates::build_date_object( $selected_start_datetime )->format( $date_compact_format );
+$selected_end_date_mobile   = Dates::build_date_object( $selected_end_datetime )->format( $date_compact_format )
 ?>
 <div class="tribe-events-c-top-bar__datepicker">
 	<button
-		class="tribe-common-h2 tribe-common-h3--min-medium tribe-common-h--alt tribe-events-c-top-bar__datepicker-button"
+		class="tribe-common-h3 tribe-common-h--alt tribe-events-c-top-bar__datepicker-button"
 		data-js="tribe-events-top-bar-datepicker-button"
 	>
 		<?php if ( $is_now ) : ?>
 			<?php esc_html_e( 'Now', 'the-events-calendar' ); ?>
 		<?php else: ?>
 			<time datetime="<?php echo esc_attr( date_i18n( 'Y-m-d', $selected_start_datetime ) ); ?>">
-				<?php echo esc_html( $selected_start_date_label ); ?>
+				<span class="tribe-events-c-top-bar__datepicker-date-mobile">
+					<?php echo $selected_start_date_mobile ?>
```

Comment:
Even with being our own, it comes from the DB, so we need to be 100% sure. No reason not to.